src/components/cc-env-var-editor-json/cc-env-var-editor-json.js

```diff
@@ -157,12 +157,12 @@ export class CcEnvVarEditorJson extends LitElement {
       ${this._errors.length > 0 ? html`
         <div class="error-list">
           ${this._errors.map(({ msg, isNotice }) => html`
```

Comment:
The `isNotice` becomes confusing. We could replace it by the intent directly or `isWarning`.
By the way, the `this._errors` is of type `ParseError` which does not describe the `isNotice` property. Maybe its the right moment to fix that.
Also, the `line` property of the `ParseError` should be optional.